.github/workflows/release-patch.yml

```diff
@@ -103,5 +104,5 @@ jobs:
 
               await releasePatchFor(inputVersion, isValidOverride ? overrideSha : undefined);
             } else { // scheduled release of AUTO_RELEASE_VERSIONS
-              await Promise.all(AUTO_RELEASE_VERSIONS.map(releasePatchFor));
+              await Promise.all(AUTO_RELEASE_VERSIONS.map(version => releasePatchFor(version)));
```

Comment:
this was passing the array index 1 as the second argument

<img width="481" height="123" alt="Image" src="https://github.com/user-attachments/assets/98d44ed0-43ab-4c48-abc3-6aa21520e5d9" />